### PR TITLE
[MERGE] GE06 Module Finished

### PR DESCRIPTION
**Pull request by vmac-odoo**
# Features
- Lots created automatically based on the VIN

### CODE REVIEW

File: ge06_team02/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "GE06: Generate Serial Number as a VIN",
+    "summary": "Generate Serial Number as a VIN",
+    "description": """
+    Motorcycle GE06_team02""",
+    "version": "0.1",
+    "category": "Kauil/Registry",
+    "license": "OPL-1",
+    "depends": ["motorcycle_registry", "stock"],
+    "data": [],
```
Comment (ayhp-odoo) [inline ge06_team02/__manifest__.py:10]: Remove if it's empty

File: ge06_team02/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "GE06: Generate Serial Number as a VIN",
+    "summary": "Generate Serial Number as a VIN",
+    "description": """
+    Motorcycle GE06_team02""",
+    "version": "0.1",
```
Comment (ayhp-odoo) [inline ge06_team02/__manifest__.py:6]: version should be 1.0.0

File: ge06_team02/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "name": "GE06: Generate Serial Number as a VIN",
+    "summary": "Generate Serial Number as a VIN",
+    "description": """
+    Motorcycle GE06_team02""",
```
Comment (ayhp-odoo) [inline ge06_team02/__manifest__.py:5]: Please be more explicit with what does this module do

File: ge06_team02/models/stock_lot.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
+from odoo import models, fields, api
```
Comment (ayhp-odoo) [inline ge06_team02/models/stock_lot.py:1]: The imports should be in alhabetica order

**Review by ayhp-odoo — CHANGES_REQUESTED** [review]

File: ge06_team02/__manifest__.py (hunk 1 above)
Comment (vmac-odoo) [reply ge06_team02/__manifest__.py]: Solved Issue

File: ge06_team02/__manifest__.py (hunk 2 above)
Comment (vmac-odoo) [reply ge06_team02/__manifest__.py]: Solved Issue

File: ge06_team02/__manifest__.py (hunk 3 above)
Comment (vmac-odoo) [reply ge06_team02/__manifest__.py]: Solved Issue

File: ge06_team02/models/stock_lot.py (hunk 4 above)
Comment (vmac-odoo) [reply ge06_team02/models/stock_lot.py]: Solved Issue

**Review by ayhp-odoo — APPROVED** [review]

**Pull request merged**